frontend/javascripts/admin/organization/organization_cards.tsx

```diff
@@ -0,0 +1,338 @@
+                strokeColor={hasExceededUserLimit ? redStrokeColor : greenStrokeColor}
+                status={hasExceededUserLimit ? "exception" : "active"}
+              />
+            </Row>
+            <Row justify="center">Users</Row>
+          </Card>
+        </Col>
+        <Col>
+          <Card actions={upgradeStorageAction}>
+            <Row style={{ padding: 20 }}>
+              <Progress
+                type="dashboard"
+                percent={usedStoragePercentage}
+                format={() => storageLabel}
+                strokeColor={hasExceededStorageLimit ? redStrokeColor : greenStrokeColor}
+                status={hasExceededStorageLimit ? "exception" : "active"}
+              />
+            </Row>
+            <Row justify="center">Storage</Row>
+          </Card>
+        </Col>
+        <Col>
+          <Card actions={upgradePlanAction}>
+            <Row justify="center" align="middle" style={{ minHeight: 160, padding: "25px 35px" }}>
+              <h3>{organization.pricingPlan}</h3>
+            </Row>
+            <Row justify="center">Current Plan</Row>
+          </Card>
+        </Col>
+      </Row>
+    </>
+  );
+}
+
+function PlanExceededAlert({ organization }: { organization: APIOrganization }) {
+  const hasPlanExpired = Date.now() > organization.paidUntil;
+
+  const message = hasPlanExpired
+    ? "Your webKnossos plan has expired. Renew your plan now to avoid being downgraded and lose access to features and prevent users from being blocked."
+    : "Your organization is using more users or storage space than included in your current plan. Upgrade now to avoid your account from being blocked.";
+  const actionButton = hasPlanExpired ? (
+    <Button
+      size="small"
+      type="primary"
+      onClick={() => UpgradePricingPlanModal.extendPricingPlan(organization)}
+    >
+      Extend Plan Now
+    </Button>
+  ) : (
+    <Button
+      size="small"
+      type="primary"
+      onClick={() => UpgradePricingPlanModal.upgradePricingPlan(organization)}
+    >
+      Upgrade Now
+    </Button>
+  );
+
+  return (
+    <Alert
+      showIcon
+      type="error"
+      message={message}
+      action={actionButton}
+      style={{ marginBottom: 20 }}
+    />
+  );
+}
+
+export function PlanAboutToExceedWarning({
+  organization,
+  usedStorageSpace,
+}: {
+  organization: APIOrganization;
+  usedStorageSpace: number;
+}) {
+  const alerts = [];
+  const isAboutToExpire =
+    moment.duration(moment(organization.paidUntil).diff(moment())).asWeeks() <= 6;
+
+  const storageWarningThresholdMB = PricingPlanEnum.Free ? 5000 : 10000;
+  const isAboutToExceedStorage =
+    usedStorageSpace >= organization.includedStorage - storageWarningThresholdMB;
+
+  if (isAboutToExpire)
+    alerts.push({
+      message:
+        "Your webKnossos plan is about to expire soon. Renew your plan now to avoid being downgraded and lose access to features and prevent users from being blocked.",
```

Comment:
```suggestion
        "Your webKnossos plan has expired. Renew your plan now to avoid being downgraded, users being blocked, and losing access to features.",
```

Reply:
Das stimmt halt nicht. Der Alert wird nur angezeigt bevor der Account ausläuft/überläuft.

Reply:
edit